### PR TITLE
Add `volumeMounts` and `securityContext` for container spec

### PR DESCRIPTION
**Pull request by thomaswanghz**

### CODE REVIEW

**Comment by thomaswanghz** [comment]
@idealhack @geckofu , please have a review, thanks.

File: jsonnetlib/k8s.jsonnet [hunk 1]
```diff
@@ -304,6 +304,8 @@ cfg {
     envmap={},
     container={},
     volumes=[],
+    volumeMounts=[],
```
Comment (idealhack) [inline jsonnetlib/k8s.jsonnet:307]: you can use [`container.volumeMounts`](https://github.com/theplant/plantbuild/blob/master/example/k8s/tests/test_volumes.jsonnet) for this

File: jsonnetlib/k8s.jsonnet [hunk 2]
```diff
@@ -304,6 +304,8 @@ cfg {
...
+    securityContext={},
```
Comment (idealhack) [inline jsonnetlib/k8s.jsonnet:308]: you can use [`podSpec.securityContext`](https://github.com/theplant/plantbuild/blob/master/example/k8s/tests/test_pod_spec.jsonnet) for this

**Comment by thomaswanghz** [comment]
switch to another way.

**Pull request closed without merging**